python/cloudbuild.yaml

```diff
@@ -13,33 +13,47 @@
 # limitations under the License.
 
 steps:
-  - name: python:3.7
-    id: 'run-unit-tests'
+  - id: 'Unit testing'
```

Comment:
Could you please update the module's README section about running a build manually via command line? Given that it now needs to be called from a specific folder, etc